client/src/app/users/add-user.component.ts

```diff
@@ -72,7 +72,7 @@ export class AddUserComponent implements OnInit {
       // Since this is for a company, we need workers to be old enough to work, and probably not older than 200.
       age: new FormControl('', Validators.compose([
         Validators.required,
-        Validators.pattern('^[0-9]+[0-9]?'),
+        Validators.pattern('^[0-9]+$'),
```

Comment:
I think the idea here was that they should be at least two digits but that is obviously captured by the min and max validation.

Reply:
Yeah, that seems like what they were going for :)